test/controllers/api/v2/parameters_controller_test.rb

```diff
@@ -21,6 +21,19 @@ def setup
     assert_not_nil assigns(:parameters)
     parameters = ActiveSupport::JSON.decode(@response.body)
     assert_not_empty parameters
+    assert_equal parameters['total'], 1
+  end
+
+  test "should get all parameters of a specific domain" do
+    mydomain = domains(:mydomain)
+    FactoryBot.create(:domain_parameter, :domain => mydomain, :name => 'additional_parameter', :value => '123')
+    get :index, params: { :domain_id => mydomain.to_param, :per_page => 'all' }
+    assert_response :success
+    assert_not_nil assigns(:parameters)
+    parameters = ActiveSupport::JSON.decode(@response.body)
+    assert_not_empty parameters
+    assert_equal parameters['total'], 2
```

Comment:
Arguably this doesn't help much in this test, because total doesn't look at pagination. But it doesn't hurt either.